app/scripts/lib/app-start.js

```diff
@@ -507,15 +507,19 @@ function (
      * Sets a UUID value that is unrelated to any account information.
      * This value is useful to determine if the logged out user qualifies for ab testing or metrics.
      */
-    initializeUuid: function () {
+    initializeUniqueUserId: function () {
       var storage = Storage.factory('localStorage', this._window);
 
       // we reuse the same uuid if it is available in local storage.
+      //
+      // Note from stomlinson, I changed the internal name to `uniqueUserId`
+      // from `uuid`, but kept `uuid` in localStorage so that a localStorage
+      // migration was not needed.
       if (storage.get('uuid')) {
```

Comment:
Seems reasonable to leave it as it is, I dont think there is much benefit in migrating the localstorage code. Also, are we using the uuid elsewhere? Cause then, changing localStorage might result in errors thrown at other places.